.github/workflows/python-publish.yml

```
@@ -82,7 +82,7 @@ jobs:
       shell: bash
       run: echo "CIBW_BUILD=cp${PYTHON_VERSION/./}-*" >> $GITHUB_ENV
     - name: Build wheels
-      uses: pypa/cibuildwheel@v2.23.3
+      uses: pypa/cibuildwheel@v3.0.0
```

```suggestion
      uses: pypa/cibuildwheel@v2.23.3
```
Let's revert this change so that the other changes can be merged.
Once that has been done, dependabot will create a separate targeted PR for this change as discussed at https://github.com/ArduPilot/pymavlink/pull/1072#issuecomment-3023588511